website/pages/case-studies/add-your-case-study.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Add Your Case Study"
+date: 2032/06/07
```

Comment:
```suggestion
date: 2022/01/01
```

I think this should in fact be the last entry, not the first :) Otherwise it's a bit weird to land on the "case studies" page and the first thing you see is not a case study, but rather a request for case studies. Later we can make it a link that looks different from the rest, for now probably easiest to just keep it as the last entry.

Reply:
Thinking about it again, I think it's also a bit strange that it says "Read the case study ->" under the li
![Screenshot 2023-04-27 at 15 35 59](https://user-images.githubusercontent.com/1121616/234896633-61401ada-c3c1-45dd-867f-beb8d7e14175.png)
nk

It would probably be best to keep this as a separate link outside of the index, so we don't have this